styles.css

```diff
@@ -458,3 +469,414 @@ form {
+  }
+
+  .headline {
+    background-image: none;
+    padding: 15%;
+    padding-top: 12%;
+    padding-bottom: 0;
+    border-radius: 0 0 0 180px;
+    height: 81vh;
+    justify-content: start;
+    width: 70vw;
+  }
+
+  .headline > * {
+    padding: 12px;
+  }
+
+  .wrapper-container {
+    margin: 0;
+    padding: 0;
+  }
+
+  /* work section */
+  .project {
+    padding: 9%;
+    padding-bottom: 0;
+    height: 60vh;
+    width: 82vw;
+  }
+
+  .container {
+    border: 1px solid #dfe1e6;
+    grid-template-columns: repeat(auto-fill, minmax(50%, 1fr));
+    grid-template-rows: repeat(6, 1fr);
+    grid-gap: 0;
+    height: 53vh;
+    padding: 2rem;
+  }
+
+  .project-1 {
+    order: 3;
+  }
+
+  .project-2 {
+    order: 4;
+    padding-bottom: 10%;
+  }
+
+  .project-3 {
+    order: 1;
+  }
+
+  .project-3 > .container {
+    width: 75vw;
+  }
+
+  .project-4 {
+    order: 2;
+  }
+
+  .container > * {
+    margin: 0;
+    padding-left: 2vw;
+  }
+
+  .container > img {
+    width: 36.7vw;
+    grid-row: 1 / 7;
+    padding: 3%;
+    padding-right: 0;
+  }
+
+  .image-desktop,
+  .li-desktop,
+  .project-desktop,
+  .client-name-desktop,
+  .name-desktop {
+    display: block;
+  }
+
+  .li-mobile,
+  .project-mobile,
+  .client-name-mobile,
+  .image-mobile {
+    display: none;
+  }
+
+  .container > .flip-img {
+    grid-column: 2 / 3;
+  }
+
+  .client-name {
+    height: 50%;
+    align-self: end;
+    margin: 2%;
+    font-size: 2.78vw;
+  }
+
+  .container > ul {
+    margin: 7%;
+    margin-right: 0;
+    height: 3vh;
+    margin-top: 0;
+    margin-bottom: 0;
+    align-self: center;
+  }
+
+  .container > ul > .name {
+    font-size: 1.25vw;
+  }
+
+  .about-project {
+    margin: 2%;
+  }
+
+  .buttons {
+    margin: 2%;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .see-project {
+    margin-left: 7%;
+    padding: 0 1% 0 1%;
+    width: 10vw;
+    align-self: start;
+    font-size: 1.39vw;
+  }
+
+  .contact {
+    width: 50%;
+  }
+
+  .logo,
+  .about-project {
+    font-size: 1.112vw;
+    line-height: 1.7;
+    text-decoration: none;
+  }
+
+  .greeting {
+    font-size: 3.334vw;
+    grid-row: 2/3;
+  }
+
+  .container > ul > li {
+    font-size: 1.2vw;
+    font-weight: 50;
+    margin: 0;
+  }
+
+  .second-text {
+    grid-row: 4/6;
+  }
+
+  .buttons > ul > li {
+    margin: 2%;
+    font-size: 0.85vw;
+    font-weight: 50;
+    padding: 0.07rem 0.3rem 0.07rem 0.3rem;
+    border-radius: 0.3rem;
+  }
+
+  .buttons > ul {
+    margin: 0;
+  }
+
+  .dropdown-content > a > img {
+    padding: 0;
+    width: 4.5vw;
+    grid-row: 1/2;
+    grid-column: 1/2;
+  }
+
+  .socials > ul > li > a > img {
+    width: 1.4vw;
+  }
+
+  /* about myself */
+  #headline-two {
+    display: grid;
+    grid-template-columns: repeat(2, 40% 60%);
+    grid-template-rows: repeat(9, 5vh);
+    height: 55vh;
+    padding: 8.2% 8.2% 4.2% 8.2%;
+    background-color: #fff;
+    border-radius: 0 20% 0 0;
+    width: 86vw;
+  }
```

Comment:
- This section of your project is quite different from the Figma design. Kindly make the necessary adjustment so it can look like the Figma as much as possible.

Your design
![image](https://github.com/agneta1991/Portfolio/assets/89091822/7aaee4b4-0f98-4561-85f7-97dcacf3f966)

Figma design
![image](https://github.com/agneta1991/Portfolio/assets/89091822/a8838ee8-441c-4639-adfd-34a92cdbf314)